content/en/docs/reference/scheduling/config.md

```diff
@@ -245,6 +247,8 @@ beware of the following changes, if applicable:
 - `DefaultPodTopologySpread` [plugin](#scheduling-plugins) is renamed to `SelectorSpread`.
 - `Unreserve` extension point is removed from Profile definition. All `Reserve`
   plugins implement an `Unreserve` call.
+- `.disablePreemption` was removed. Users can disable preemption by disabling the
+  "DefaultPreemption" PostFilter plugin.
```

Comment:
This detail should go into a changelog.

If it's very important to document how to disable preemption, I'd add a new task page. However, it doesn't seem important to document that specifically as part of the Concept section page on the scheduling framework.